app.rb

```diff
@@ -0,0 +1,137 @@
+require './book'
+require './person'
+require './rental'
+require './student'
+require './teacher'
+require './classroom'
+
+class App
+  def initialize(menu)
+    @menu = menu
+    @people = []
+    @books = []
+    @rentals = []
+  end
+
+  def list_all_books
+    puts 'There are no books yet! Kindly add books.' if @books.empty?
+
+    @books.each { |book| puts "Title: #{book.title}, Author: #{book.author}" }
+    sleep 0.75
+    @menu.display_menu
+  end
+
+  def list_all_people
+    puts 'There are no people yet! Kindly add a student or teacher.' if @people.empty?
+    @people.map { |person| puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}" }
+    sleep 0.75
+    @menu.display_menu
+  end
+
+  def create_person
+    print 'Do you want to create a student (1) or teacher (2) [Input a number]: '
+    option = gets.chomp
+
+    case option
+    when '1'
+      create_student
+    when '2'
+      create_teacher
+    else
+      puts 'Invalid input. kindly type 1 or 2'
+    end
+    @menu.display_menu
+  end
+
+  def create_student
+    print 'Age: '
+    age = gets.chomp.to_i
+
+    print 'Name: '
+    name = gets.chomp
+
+    print 'Has parent permission? [Y/N]: '
+    parent_permission = gets.chomp.downcase
+
+    print 'Classroom: '
+    classroom = gets.chomp
+
+    student = Student.new(age, classroom, name, parent_permission)
+    @people.push(student)
+
```

Comment:
- [ ] Very great work so far. I tried to add a new person, student but I got an error after I input the classroom option. See the error below - 

**Error**
![image](https://user-images.githubusercontent.com/84156714/216773466-9f143141-1826-4e4d-8356-d14c9952f6a2.png)

I looked closely into the video sample for this project, i did not see _**classroom**_ as an option when creating a student. It means the Student class parameter, **_classroom_** should or may have a default value set, or hard coded in the class. Please find a suitable way to fix this error.

Reply:
The classroom error has been fixed. Thank you so much